selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/utils/WaitUtils.java

```diff
@@ -46,4 +54,67 @@ public static void sleepQuietly(int timeout, TimeUnit timeUnit) {
       throw new RuntimeException(e);
     }
   }
+
+  /**
+   * Waits during {@code timeout} until {@code condition} has a "true" state.
+   *
+   * @param condition expression which should be performed
+   * @param timeout waiting time
+   * @param delayBetweenAttemptsInMilliseconds delay between tries of {@code condition} execution in
+   *     milliseconds
+   */
+  public static void waitSuccessCondition(
+      BooleanSupplier condition,
+      int timeout,
```

Comment:
it's better to declare implicitely that timeout is in milliseconds itself, by naming - `timeoutInMillisec`